### ISSUE TITLE
"Getting started" section is incomplete/out-of-date

### ISSUE THREAD

**Issue by ptkw** [transpect/transpect.github.io#1]
Following the **Getting started** tutorial as written does not lead to a working example of transpect.

In addition to the problem with the `unzip-extension` module, as described in [this issue](https://github.com/transpect/unzip-extension/issues/6), the tutorial doesn't mention the need to install the `htmlreports` module. Without it, executing `calabash` produces the following error message:
`$ ./calabash/calabash.sh -o result=MyXMLfile.xml docx2hub/xpl/docx2hub.xpl docx=MyWordfile.docx`
`ERROR: docx2hub/xpl/docx2hub.xpl:113:74:err:XS0052:Cannot import: http://transpect.io/htmlreports/xpl/errorPI2svrl.xpl`
`ERROR:     cause: I/O error reported by XML parser processing http://transpect.io/htmlreports/xpl/errorPI2svrl.xpl: http://transpect.github.io/htmlreports/xpl/errorPI2svrl.xpl`
`ERROR: It is a static error if the URI of a p:import cannot be retrieved or if, once retrieved, it does not point to a p:library, p:declare-step, or p:pipeline.`
`ERROR: Underlying exception: I/O error reported by XML parser processing http://transpect.io/htmlreports/xpl/errorPI2svrl.xpl: http://transpect.github.io/htmlreports/xpl/errorPI2svrl.xpl`

Cloning the `htmlreports` module (and adding it to the xmlcatalog) solves the issue.

**Comment by gimsieke** [comment]
That’s right. This requirement crept in a couple of weeks ago. We’ll update the tutorial. The complete [docx2jats-demo](https://github.com/transpect/docx2jats-demo) should continue to work though, since all submodules have been attached with a fixed revision, as of the time when we created the demo.

**Comment by ptkw** [comment]
Thank you for your prompt reply. The error message is self-explanatory enough, but I thought I'll mention the issue anyway since it might be potentially discouraging for new transpect users visiting the website.
